app/models/user.rb

```diff
@@ -1,10 +1,8 @@
-# frozen_string_literal: true
-
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :validatable
   has_many :categories, dependent: :destroy
-  has_many :transactions, dependent: :destroy
+  # has_many :transacs, dependent: :destroy
```

Comment:
- [ ] The user can have many transactions. Please fix the association between `users` and `transactions`.